doc/source/data/loading-data.rst

```diff
@@ -35,7 +35,7 @@ Generating Synthetic Data
       .. doctest::
 
         >>> import ray
-        >>> ds = ray.data.range_tensor(100 * 64 * 64, shape=(64, 64))
+        >>> ds = ray.data.range_tensor(1, shape=(64, 64))
```

Comment:
Ray Data does not preserve order by default. So to guarantee the output below, we use a Dataset of size 1.

Reply:
Can we enable determinism with an environment variable?